src/ngx_http_lua_variable.c

```diff
@@ -364,21 +364,24 @@ ngx_http_lua_ffi_var_get(ngx_http_request_t *r, u_char *name_data,
 int
 ngx_http_lua_ffi_var_set(ngx_http_request_t *r, u_char *name_data,
     size_t name_len, u_char *lowcase_buf, u_char *value, size_t value_len,
-    u_char *errbuf, size_t errlen)
+    u_char *errbuf, size_t *errlen)
 {
     u_char                      *p;
     ngx_uint_t                   hash;
     ngx_http_variable_t         *v;
     ngx_http_variable_value_t   *vv;
     ngx_http_core_main_conf_t   *cmcf;
 
+    /* we don't return actual errlen so need to NUL-terminate it */
```

Comment:
I'm afraid this comment is now obsolete since we do return the actual errmsg length.

Reply:
removed